pkg/volume/aws_ebs/aws_util.go

```diff
@@ -178,7 +182,7 @@ func verifyAllPathsRemoved(devicePaths []string) (bool, error) {
 // Returns list of all paths for given EBS mount
 // This is more interesting on GCE (where we are able to identify volumes under /dev/disk-by-id)
 // Here it is mostly about applying the partition path
-func getDiskByIdPaths(partition string, devicePath string) []string {
+func getDiskByIdPaths(volumeID aws.KubernetesVolumeID, partition string, devicePath string) []string {
```

Comment:
Not a blocker but what we should do is - update the device path returned by this method in volume_manager's actual state of world, so as subsequent functions have correct device path if they need. I scanned the code quickly though and most interesting things happen in `MountVolume` function and there we use devicePath returned by this function for everything (stored in a local variable) and hence we should be fine.